dependency/catalog_node.go

```diff
@@ -152,5 +154,5 @@ type NodeServiceList []*NodeService
 func (s NodeServiceList) Len() int      { return len(s) }
 func (s NodeServiceList) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s NodeServiceList) Less(i, j int) bool {
-	return s[i].Service <= s[j].Service
```

Comment:
@williambailey I'm mostly worried about backwards-compatability. I don't want template rendering (order) to change if a user updates Consul Template as that would be unexpected. Can we leave service in there for now and maybe we can remove it for the 1.0 release.